src/docs_website/src/html/page-script.js

```diff
@@ -78,14 +76,13 @@ function syncSideNavWithLocation() {
       if (a.href.endsWith(path)) {
         a.classList.add("target");
         for (let parent = a.parentElement; parent; parent = parent.parentElement) {
-          if (parent.matches(".menu")) {
-            const head = parent.previousSibling;
-            if (head?.matches?.(".menu-head")) head.classList.add("expanded");
-          } else if (parent.matches(".menu-head")) {
-            parent.classList.add("expanded"); // expand for discoverability
+          if (parent.tagName === "DETAILS") {
+            parent.open = true;
           }
         }
       }
     });
   }
-}
\ No newline at end of file
+}
+
+syncSideNavWithLocation();
```

Comment:
ditto here. All our source-files should end in `\n`, for consistency and diff minimization.